lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1201,6 +1220,63 @@ LLVMToSPIRV::transSpcvCast(CallInst* CI, SPIRVBasicBlock *BB) {
 }
 
 SPIRVValue *
+LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II, SPIRVBasicBlock *BB) {
+  auto getMemoryAccess = [](MemIntrinsic *MI)->std::vector<SPIRVWord> {
+    std::vector<SPIRVWord> MemoryAccess(1, MemoryAccessMaskNone);
+    if (SPIRVWord AlignVal = MI->getAlignment()) {
+      MemoryAccess[0] |= MemoryAccessAlignedMask;
+      MemoryAccess.push_back(AlignVal);
+    }
+    if (MI->isVolatile())
+      MemoryAccess[0] |= MemoryAccessVolatileMask;
+    return MemoryAccess;
+  };
+
+  switch (II->getIntrinsicID()) {
+  case Intrinsic::fmuladd : {

```

Comment:
We don't have immediate need for this right now, do we? Besides there is no facilities for emitting warnings. This commit is not intended to cover all llvm intrinsics, I just wanted to fix #174 and llvm.memset which revealed during testing. Probably we should submit another issue to handle llvm.fma.*